openedx/tutor-contrib-aspects
# feat: Add support ClickHouse clusters

- Update database / user / grant init to support cluster
- Updates migrations to support cluster
- Updates settings to add new cluster name
- Remove dbt-specific cluster name

Tested against an Altinity Cloud cluster:

- Init works
- dbt works
- Ralph works
- Vector works
- Superset reports work
- xapi-db-load works

Testing instructions:

- Set up a ClickHouse cluster (Clickhouse Cloud or Altinity Cloud are easiest but Clickhouse Cloud needs special handling)
- Update your `CLICKHOUSE_HOST` to match the cluster hostname
- Update your `CLICKHOUSE_SECURE_CONNECTION` to true if necessary (if the host is HTTPS)
- Update your `CLICKHOUSE_CLUSTER_NAME` to the name of the cluster
- Save your tutor config
- Do init as necessary for your setup
- Test Aspects as usual, generate some events, do xapi-db-load, etc.
- Events should appear in Superset reports
